app/Moldova/Service/Contracts.php

```diff
@@ -118,8 +118,7 @@ public function getGoodsAndServices($type, $limit, $year, $condition = '', $colu
     public function getTotalContractAmount()
     {
         $total = $this->contracts->getTotalContractAmount();
-
-        return $total[0]['amount'];
+        return isset($total[0]) ? $total[0]['amount'] : '';
```

Comment:
If the amount doesn't exist in $total, $total[0]['amount'] will return 'null'. If $total is null, $total[0] was causing error. I have refactored the code and we don't have to check for the $total[0] index.